PowerEditor/installer/nativeLang/slovak.xml

```diff
@@ -978,17 +1052,17 @@ Zadať môžete viacero značiek pomocou medzery na oddelenie rôznych čísel."
                 <Item id="7004" name="Usporiadať karty"/>
             </Window>
             <ColumnEditor title="Stĺpcový editor">
-                <Item id="2023" name="Vložiť text"/>
-                <Item id="2033" name="Vložiť číslo"/>
-                <Item id="2030" name="Začiatočné číslo:"/>
-                <Item id="2031" name="Zvyšovať po:"/>
-                <Item id="2035" name="Nuly na začiatku"/>
-                <Item id="2036" name="Počet opakovaní:"/>
-                <Item id="2032" name="Formát"/>
-                <Item id="2024" name="Dec"/>
-                <Item id="2025" name="Oct"/>
-                <Item id="2026" name="Hex"/>
-                <Item id="2027" name="Bin"/>
+                <Item id="2023" name="Vložiť &amp;text"/>
+                <Item id="2033" name="Vložiť čí&amp;slo"/>
+                <Item id="2030" name="&amp;Začiatočné číslo:"/>
+                <Item id="2031" name="Z&amp;vyšovať po:"/>
+                <Item id="2035" name="&amp;Nuly na začiatku"/>
+                <Item id="2036" name="&amp;Počet opakovaní:"/>
+                <Item id="2032" name="&amp;Formát"/>
```

Comment:
Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;` in front of 'F' should be removed.